src/main/java/ezschedule/model/Model.java

```diff
@@ -64,43 +64,53 @@ public interface Model {
     void setScheduler(ReadOnlyScheduler scheduler);
 
     /**
-     * Returns true if an event with the same identity as {@code event} exists in the Scheduler.
+     * Returns true if an event with the same identity as {@code event} exists in the scheduler.
      */
     boolean hasEvent(Event event);
 
     /**
-     * Returns true if another event exists at the given time in the Scheduler.
+     * Returns true if another event exists at the same time as {@code event} in the scheduler.
```

Comment:
use `{@code Event}` and `{@code Scheduler}` instead?

Reply:
Ok will update for next iteration